redaxo/src/addons/media_manager/lang/de_de.lang

```diff
@@ -34,15 +34,19 @@ media_manager_type_caption = Liste der angelegten Mediatypen
 media_manager_type_no_types = Keine Mediatypen vorhanden
 media_manager_type_create = Mediatyp erstellen
 media_manager_type_edit = editieren
+media_manager_type_copy = kopieren
```

Comment:
> was haltet ihr von "duplizieren" anstatt kopieren?

kommt darauf was genau passieren wird.

Duplizieren > speichert eine Kopie des Datensatzes direkt in der DB oder im Filesystem
Kopieren > legt eine Kopie des Datensatzes in den Zwischenspeicher, hier bei uns wäre es wohl das Formular im Editmodus.